sor/src/main/java/com/bazaarvoice/emodb/sor/core/DefaultDataStore.java

```diff
@@ -758,7 +758,7 @@ public void afterWrite(Collection<RecordUpdate> updateBatch) {
                 // Update the audit to include the tags associated with the update
                 updateBatch.forEach(update -> {
                     Audit augmentedAudit = AuditBuilder.from(update.getAudit())
-                            .set(Audit.SHA1, Hashing.sha1().hashUnencodedChars(update.getDelta().toString()).toString())
+                            .set(Audit.SHA1, Hashing.sha512().hashUnencodedChars(update.getDelta().toString()).toString())
```

Comment:
needs to be reverted (and the check can suppressed)

Reply:
the changes were reverted